<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1128</width>
    <height>815</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Plotting Sift Points</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="GLWidget" name="widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Show:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkInliers">
           <property name="text">
            <string>Inliers</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkMatch">
           <property name="text">
            <string>Match</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string># Point Corresps:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="textPCorresp">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string># Inliers:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="textInliers">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Homography:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <item row="0" column="0">
            <widget class="QTextEdit" name="m11">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QTextEdit" name="m12">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QTextEdit" name="m13">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QTextEdit" name="m21">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QTextEdit" name="m22">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QTextEdit" name="m23">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QTextEdit" name="m31">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QTextEdit" name="m32">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QTextEdit" name="m33">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Point:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="textPid">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="textPx">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="textPy">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Corresp Point:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="textPCx">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="textPCy">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="btBackward">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
             <property name="text">
              <string>|&lt;</string>
             </property>
             <property name="iconSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btBackwardOne">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
             <property name="text">
              <string>&lt;&lt;</string>
             </property>
             <property name="iconSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btPlay">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
             <property name="text">
              <string>&gt;</string>
             </property>
             <property name="iconSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btStop">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
             <property name="text">
              <string>||</string>
             </property>
             <property name="iconSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btForwardOne">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
             <property name="text">
              <string>&gt;&gt;</string>
             </property>
             <property name="iconSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btForward">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
             <property name="text">
              <string>&gt;|</string>
             </property>
             <property name="iconSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1128</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load video</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save info</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>GLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
